### PR TITLE
Add support for EXTERNAL_PORT and ENV_FILE to Docker Entrypoint.

### PR DESCRIPTION
**Pull request by bartoszjedrzejewski**

### CODE REVIEW

File: docker/docker_entrypoint.sh [hunk 1]
```diff
@@ -11,6 +18,7 @@ BASE_PATH=${BASE_PATH:?'Base path should be specified'}
 RPC_PORT=${RPC_PORT:-9933}
 WS_PORT=${WS_PORT:-9943}
 PORT=${PORT:-30333}
+EXTERNAL_PORT=${EXTERNAL_PORT:-${PORT}}
```
Comment (Jamakase) [inline docker/docker_entrypoint.sh:21]: I see that you are assigning `EXTERNAL_PORT` but where is it used?

**Review by Jamakase — APPROVED** [review]

File: docker/docker_entrypoint.sh [hunk 2]
```diff
@@ -1,6 +1,13 @@
 #!/usr/bin/env bash
 set -euo pipefail
 
+# load env variables from a file if ENV_FILE is set
+if [[ -n "${ENV_FILE:-}" ]] && [[ -f "${ENV_FILE}" ]]; then
```
Comment (Jamakase) [inline docker/docker_entrypoint.sh:5]: What is the expected behaviour if both env file and env variables are set? Should we prefer file one over env? Or fail?

Comment (bartoszjedrzejewski) [reply docker/docker_entrypoint.sh:5]: Expected behaviour is that env file overrides previously set env variables. For now this is what is needed for: https://github.com/Cardinal-Cryptography/aleph-infra/blob/main/initContainer/entrypoint.sh there's no overlap currently.

File: docker/docker_entrypoint.sh (hunk 1 above)
Comment (bartoszjedrzejewski) [reply docker/docker_entrypoint.sh:21]: https://github.com/Cardinal-Cryptography/aleph-infra/blob/main/initContainer/entrypoint.sh this is where it's used. External port can be different than port variable while setting up a public-addr for testnet for example.

**Review by skrolikowski-10c — APPROVED** [review]

**Pull request merged**